src/main/java/com/solvd/carina/demo/gui/pages/desktop/zebrunner/ZebRunnerHomePage.java

```diff
@@ -0,0 +1,137 @@
+package com.solvd.carina.demo.gui.pages.desktop.zebrunner;
+
+import com.solvd.carina.demo.gui.components.zebrunner.header.HeaderMenu;
+import com.solvd.carina.demo.gui.pages.common.ZebRunnerHomePageBase;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.List;
+import java.util.Objects;
+import java.util.Random;
+
+public class ZebRunnerHomePage extends ZebRunnerHomePageBase {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ZebRunnerHomePage.class);
+    @FindBy(css = "header.md-header")
+    private HeaderMenu header;
+    @FindBy(xpath="//nav[@class='md-nav md-nav--primary']")
+    private ZebRunnerNavigationMenu navigationMenu;
```

Comment:
```suggestion
    @FindBy(css = "header.md-header")
    private HeaderMenu header;
    @FindBy(xpath="//nav[@class='md-nav md-nav--primary']")
    private ZebRunnerNavigationMenu navigationMenu;
```
You had a different set of formatting standards in other files. Unify your code style.

Reply:
Fixed the formatting issues